res/controllers/Gemini-G4V-scripts.js

```diff
@@ -0,0 +1,1802 @@
+/**
+ * Gemini G4V controller script
+ *
+ * For Mixxx 2.0.0+
+ * Written by Javier Vilarroig 2018
+ *
+ **/
+
+/*
+ * Mapping architecture
+ *
+ * Class controller - The full controller
+ *     Contains:
+ *         - decks
+ *         - common controls (not in a deck)
+ *         - common leds (not in a deck)
+ *         - common levels (not in a deck)
+ * Class deck - One deck, including the mixer
+ *     Contains:
+ *         - deck controls
+ *         - deck leds
+ *         - deck levels
+ * Class Connection - One connection between a control and a call back
+ * Class control - One control
+ * Class led - One led
+ * Class level - One level
+ */
+
+print("Loading G4V script");
```

Comment:
I think we can remove this.

Reply:
No problem. The prints are more for debugging while  developing. I can remove them.